tests/unit/controllers/sms-gateway.js

```diff
@@ -60,42 +62,44 @@ exports['post() should update statuses supplied in request'] = function(test) {
       { id:'5', status:'FAILED', reason:'bad' },
     ],
   } };
-
   // when
-  controller.post(req, function(err) {
+  controller.post(req, err => {
     // then
-    test.equals(err, null);
-    test.equals(updateMessage.callCount, 5);
-    test.equals(updateMessage.withArgs('1', { state:'received-by-gateway' }).callCount, 1);
-    test.equals(updateMessage.withArgs('2', { state:'forwarded-by-gateway' }).callCount, 1);
-    test.equals(updateMessage.withArgs('3', { state:'sent' }).callCount, 1);
-    test.equals(updateMessage.withArgs('4', { state:'delivered' }).callCount, 1);
-    test.equals(updateMessage.withArgs('5', { state:'failed', details:{ reason:'bad' } }).callCount, 1);
+    test.equal(err, null);
```

Comment:
Yes. Equals (and any other puraly sounding functions) is actually deprecated and has been for some time. I have considered changing them everywhere in the code but I don't want to piss people off ;-)